internal/controllers/user_integrations_auth_controller.go

```diff
@@ -204,3 +210,57 @@ func (u *UserIntegrationAuthController) persistOauthCredentials(ctx context.Cont
 
 	return nil
 }
+
+// GetCommandsByIntegration godoc
+// @Description Get a list of available commands by integration
+// @Tags        user-devices
+// @Produce     json
+// @Accept      json
+// @Param       tokenID path string                   true "token id for integration"
+// @Security    ApiKeyAuth
+// @Success     200 {object} controllers.GetCommandsByIntegrationResponse
+// @Security    BearerAuth
+// @Router      /integration/:tokenID/commands [get]
+func (u *UserIntegrationAuthController) GetCommandsByIntegration(c *fiber.Ctx) error {
+	tokenID := c.Params("tokenID")
+	tkID, err := strconv.ParseUint(tokenID, 10, 64)
+	if err != nil {
+		return fiber.NewError(fiber.StatusBadRequest, "could not process the provided tokenId!")
+	}
+
+	intd, err := u.DeviceDefSvc.GetIntegrationByTokenID(c.Context(), tkID)
+	if err != nil {
+		u.log.Err(err).Str("Calling Function", "GetIntegrationByTokenID").Uint64("tokenID", tkID).Msg("Error occurred trying to get integration using tokenID")
```

Comment:
nit: we haven't been putting a space in the key value for the log context, so perhaps change for consistency? eg: .Str("callingFunction",...